modules/core/arrow-core-data/src/test/kotlin/arrow/core/OptionTest.kt

```diff
@@ -16,13 +16,7 @@ import arrow.mtl.extensions.option.traverseFilter.traverseFilter
 import arrow.syntax.collections.firstOption
 import arrow.test.UnitSpec
 import arrow.test.generators.option
-import arrow.test.laws.FunctorFilterLaws
-import arrow.test.laws.HashLaws
-import arrow.test.laws.MonadFilterLaws
-import arrow.test.laws.MonoidLaws
-import arrow.test.laws.MonoidalLaws
-import arrow.test.laws.ShowLaws
-import arrow.test.laws.TraverseFilterLaws
+import arrow.test.laws.*
```

Comment:
Ktlint will flag this as an error. You need to not keep the *

Reply:
Told my IDE to not "optimize imports"